introduction_to_amazon_algorithms/forecasting_services_comparison/sagemaker_example.ipynb

```diff
@@ -0,0 +1,1543 @@
+{
```

Comment:
<p>The pricing link is the correct one, but the link text title should be <code>Amazon SageMaker Pricing</code> rather than <code>Available SageMaker Studio Instance Types</code> .</p><p> </p><p>It's worth pointing out that there is an existing  <code>use_spot_instances</code>  parameter in the estimator training, which makes it very easy to switch to spot instances with a boolean flag.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3532/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>